### ISSUE TITLE
System requirements

### ISSUE THREAD

**Issue by csteelatgburg** [MicrosoftDocs/windowsserverdocs#4045]
The page says that the client will run on systems running macOS version 10.10 and newer, but the version 10 client requires 10.12 or higher. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0517ea6a-9417-31a8-39a5-f6128387fee5
* Version Independent ID: 7636ec70-e1dd-8c6b-516c-8af324d9f645
* Content: [Get started with the macOS client](https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/remote-desktop-mac#feedback)
* Content Source: [WindowsServerDocs/remote/remote-desktop-services/clients/remote-desktop-mac.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/remote/remote-desktop-services/clients/remote-desktop-mac.md)
* Product: **windows-server**
* Technology: **remote-desktop-services**
* GitHub Login: @lizap
* Microsoft Alias: **elizapo**

**Comment by stonkeep** [comment]
Same here, Any solution?

**Comment by josep-amer** [comment]
Same problem here. I am running 10.11.6 OS X El Capitán and cannot download version 10.3.8 of Microsoft remote desktop 10. Any advice on how to install and use a previous version of it on my computer?

**Comment by illfated** [comment]
Did you try to download earlier versions from this page,
- https://install.appcenter.ms/orgs/rdmacios-k2vy/apps/microsoft-remote-desktop-for-mac/distribution_groups/all-users-of-microsoft-remote-desktop-for-mac

... or did you try to find it in the App store?

**Comment by josep-amer** [comment]
Thanks for your message. I cannot see any earlier versions to download in the link you shared (please, see screenshot attached).

There is no other version of Microsoft Remote Desktop in the App store but the 10.3.9.

![Screen Shot 2020-04-13 at 09 44 21](https://user-images.githubusercontent.com/45817289/79102882-ed093e00-7d6b-11ea-8e3f-5f4a662f8d86.png)

**Comment by illfated** [comment]
Oh. I see. I can reproduce your issue when I use Mozilla Firefox, but not when using Google Chrome.
With the Google Chrome browser on Windows, I am able to view 41 versions since September 5, 2019.

**Comment by josep-amer** [comment]
In my case, I was only able to view and download versions using Safari (weird).

Anyway, I downloaded the earliest one, September 5, 2019, and still got the same message: "You can’t use this version of the application “Microsoft Remote Desktop Beta” with this version of OS X. You have OS X 10.11.6. The application requires OS X 10.12 or later."

**Comment by illfated** [comment]
Good point. Thanks for testing and confirming that. From this, I can only presume we need to update the docs.microsoft.com page to state that the macOS RDS client requires OS X version 10.12 or later.

**Comment by fdavis99** [comment]
download

**Comment by fdavis99** [comment]
Microsoft Remote Desktop Beta 10.2.13 works for me on Mac OS 10.11 El Capitan.

Closes version I found for download is 10.2.4 https://microsoft-remote-desktop-10.en.softonic.com/mac. Maybe its "Check for updates" will upgrade you to a later 10.2.

Microsoft: older Mac hardware (2011 and earlier) runs great, but can't upgrade to MacOS newer than 10.11. Please keep Microsoft Remote Desktop 10.2.13 available!

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
